src/components/PopularCommunity/PopularCommunity.jsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import Card from '@mui/material/Card';
+import { CardActionArea } from '@mui/material';
+import CardMedia from '@mui/material/CardMedia';
+import Typography from '@mui/material/Typography';
+import Grid from '@mui/material/Grid';
+import "./PopularCommunity.css";
+
+const PopularCommunity = (props) => {
+    const {name,alt,image}=props
+  
+    return (
+    <Card sx={{ maxWidth: 345 }} raised='true'>
```

Comment:
The code should be indented two spaces in following the indenting conventions of other files. This includes an indentation needed in line 13-32 (components inside `return ()` statement) and line 15-30 (components inside `<CardActionArea>` component)

Reply:
Thanks for the feedback Hyung, the changes have been committed :)